src/main/resources/application.yml

```diff
@@ -77,6 +81,7 @@ pdfService:
 
 feature_toggles:
   watermark_pdf: false
+  document_management: true
```

Comment:
I do wonder whether we still need that feature toggle since we go with the integration string into next release.

Reply:
hmm? never assume integrations happen, could not happen for many reasons.
Please keep the toggle till the code is live